lib/bundler/shared_helpers.rb

```diff
@@ -194,13 +194,15 @@ def ensure_same_dependencies(spec, old_deps, new_deps)
         "\nEither installing with `--full-index` or running `bundle update #{spec.name}` should fix the problem."
     end
 
-    def pretty_dependency(dep, print_source = false)
+    def pretty_dependency(dep, print_source = false, print_requirements = false)
```

Comment:
why the third parameter? I dont think we _ever want to print `(>= 0)`

Reply:
Yeah. removed it 😄